test/tests/val-2e/val-2ee.i

```diff
@@ -3,13 +3,13 @@
 # No Soret effect, or trapping included.
 
 # Surface reaction data used in TMAP7 case
-K_r_pre_D2 = '${units ${fparse 2.502e-24 / sqrt(4 * temperature)} m^4/s -> mum^4/s}'
-K_r_pre_H2 = '${units ${fparse 2.502e-24 / sqrt(2 * temperature)} m^4/s -> mum^4/s}'
-K_r_pre_HD = '${units ${fparse 2.502e-24 / sqrt(3 * temperature)} m^4/s -> mum^4/s}'
-K_r_energy = '${units ${fparse -11836 * R} J/mol}'
-K_d_D2 = "${units ${fparse 2.1897e22 / sqrt(4 * temperature)} at/m^2/Pa -> at/mum^2/Pa}"
-K_d_H2 = '${units ${fparse 2.1897e22 / sqrt(2 * temperature)} at/m^2/Pa -> at/mum^2/Pa}'
-K_d_HD = '${units ${fparse 2.1897e22 / sqrt(3 * temperature)} at/m^2/Pa -> at/mum^2/Pa}'
+K_r_pre_D2 = '${units ${fparse 2.502e-24 / sqrt(4 * 870)} m^4/s -> mum^4/s}'
+K_r_pre_H2 = '${units ${fparse 2.502e-24 / sqrt(2 * 870)} m^4/s -> mum^4/s}'
+K_r_pre_HD = '${units ${fparse 2.502e-24 / sqrt(3 * 870)} m^4/s -> mum^4/s}'
+K_r_energy = '${units ${fparse -11836 * 8.31446261815324} J/mol}'
+K_d_D2 = '${units ${fparse 2.1897e22 / sqrt(4 * 870)} at/m^2/Pa -> at/mum^2/Pa}'
+K_d_H2 = '${units ${fparse 2.1897e22 / sqrt(2 * 870)} at/m^2/Pa -> at/mum^2/Pa}'
+K_d_HD = '${units ${fparse 2.1897e22 / sqrt(3 * 870)} at/m^2/Pa -> at/mum^2/Pa}'
```

Comment:
Same comment here, we should be defining temperature and constants once.

use `.params` base and specific files that only contain the parameter values (and no block).